test/integration_test/common_test.go

```diff
@@ -98,9 +101,11 @@ func TestMain(t *testing.T) {
 	}
 	// we can either run cluster domain tests or the other tests
 	// the underlying setup of clusters is different
-	if os.Getenv("ENABLE_CLUSTER_DOMAIN_TESTS") != "" {
+	if _, err := strconv.ParseBool(os.Getenv(enableClusterDomainTests)); err == nil {
```

Comment:
You also have to check the value of the bool ie the first return value from `ParseBool`

Reply:
Fixed.